docs/application/dotnet/guides/alarm/alarms.md

```diff
@@ -56,6 +56,9 @@ To enable your application to use the alarm functionality:
 
 You can set an alarm which, when it expires, either launches an application or sends a notification to the user:
 
+> [!NOTE]
+> Since Tizen 6.0, , the time period of an alarm can be one of the values of `Tizen.Applications.AlarmStandardPeriod`. This time period value will be phase-aligned with another time period of the alarm.
```

Comment:
1. Please remove the extra comma after "Tizen 6.0".
2. Can we add "value" after "time period" in the second part:

This time period value will be phase-aligned with another time period value of the alarm.

3. As per the previous comment, https://github.com/Samsung/tizen-docs/pull/1078#discussion_r407322760. Are you sure that we don't need to elaborate further?

Reply:
I've updated the sentence, please rephrase it.